src/ILCompiler.RyuJit/src/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -1541,5 +1541,30 @@ private void getGSCookie(IntPtr* pCookieVal, IntPtr** ppCookieVal)
                 throw new NotImplementedException("getGSCookie");
             }
         }
+
+        private bool pInvokeMarshalingRequired(CORINFO_METHOD_STRUCT_* handle, CORINFO_SIG_INFO* callSiteSig)
+        {
+            // calli is covered by convertPInvokeCalliToCall
```

Comment:
Can you also move `convertPInvokeCalliToCall` here? The ReadyToRun version can just throw NotImplemented. Should be unreachable.

Reply:
Good point, forked.